src/Theme.tsx

```diff
@@ -337,8 +337,10 @@ export type Breakpoint = keyof typeof breakpoints
 /** All available type sizes */
 export type TypeSizes = typeof themeProps.typeSizes
 /** All available sizes for our sans font */
-export type SansSize = keyof TypeSizes["sans"]
+export type SansSize = keyof TypeSizes["sans"] | Array<keyof TypeSizes["sans"]>
 /** All available sizes for our serif font */
-export type SerifSize = keyof TypeSizes["serif"]
+export type SerifSize =
+  | keyof TypeSizes["serif"]
+  | Array<keyof TypeSizes["serif"]>
 /** All available sizes for our display font */
 export type DisplaySize = keyof TypeSizes["display"]
```

Comment:
The `Display` font is deprecated but its still in use in a few spots -- might as well update this too.

Reply:
Do you mean "remove line 346 because it is deprecated", or "modify line 346 to also support `| Array<keyof TypeSizes["display"]>`?

Reply:
It would be the latter.

Reply:
Cool - this is added. I don't think there are any associated changes needed outside of line 346 of Theme.tsx - let me know if I'm incorrect!